src/tool/subcommands/api_cmd.rs

```diff
@@ -1390,10 +1395,18 @@ async fn run_tests(
     while let Some(Ok((method_name, test_result))) = futures.next().await {
         let forest_status = test_result.forest_status;
         let lotus_status = test_result.lotus_status;
+        let success = match (&forest_status, &lotus_status) {
+            (TestSummary::Valid, TestSummary::Valid)
+            | (TestSummary::Timeout, TestSummary::Timeout) => true,
+            (TestSummary::Rejected(ref reason_forest), TestSummary::Rejected(ref reason_lotus))
```

Comment:
This will make the tests pass even if both inputs to Forest and Lotus are accidentally malformed, no? Do you think it'd be safer to explicitly declare tests that should (could) fail and only allow them to do so?

Reply:
Fixed.